src/Http/Http/src/Features/FormFeature.cs

```diff
@@ -71,8 +72,19 @@ internal FormFeature(HttpRequest request, FormOptions options, Endpoint? endpoin
     {
         get
         {
-            _ = MediaTypeHeaderValue.TryParse(_request.ContentType, out var mt);
-            return mt;
+			MediaTypeHeaderValue? mt = null;
+
+			if (_request is not null)
+			{
+				_ = MediaTypeHeaderValue.TryParse(_request.ContentType, out mt);
+			}
+
+			if (Form is not null && mt is null)
+			{
+				mt = _formContentType;
```

Comment:
The idea here is that there might still be a `ContentType`, even if there's no request (i.e. as long as there's a form)?

Reply:
The assumption is that if there's a form there will also always be a content type, and if there is both a request and a form, the request content type should take priority, if existing and valid. If there is no form and no request, the getter will return null.